labs/lab5/chapter5.md

```diff
@@ -0,0 +1,77 @@
+# LAB 5: OpenShift templates and web console
+
+In this lab we introduce how to simplify your container deployments w/ OpenShift templates.  We will also explore the web console.
+
+This lab should be performed on **workstation.example.com** unless otherwise instructed.
+
+Expected completion: 20 minutes
+
+## Project preparation
+
+We should still be in our "production" project space at this point.
+```shell
+$ oc project
+Using project "production" on server "https://atomic-host.example.com:8443".
+```
+
+Ensure you're still logged in as the developer user & clean up the resources deployed in chapter 4.
+```shell
+$ oc whoami
+developer
+
+$ oc delete all --all
+```
+
+Ensure the following command displays "No resources found" before proceeding.
+```shell
+$ oc get all
+No resources found.
+```
+
+## Wordpress templated deployment
+
+This time, let's simplify our deployment by creating an application template.
+
+Deploy the wordpress template file included w/ lab5:
+```shell
+$ cd ~/summit-2017-container-lab/labs/lab5/
+
+# add your template to the production project
+$ oc create -f wordpress-template.yaml
```

Comment:
so we discussed this on friday - we ended up here. basically we didn't want to respin the disk image and also wanted to give them a little something more. tommy came up with the idea of creating a template they can use to just `oc new-app`. Creating the template ourselves allows us to re-use the container images we already created in the lab. 

Since this is "lab 5" and we are probably running thin on "lab time" at this point, I think it is fair to say this is magic and but fee free to look at the template file.

Also one thing we can do is grep the template file for our image names to show the students we are using the images they just created.